docs/src/test/scala/docs/http/scaladsl/server/ExceptionHandlerExamplesSpec.scala

```diff
@@ -25,7 +25,6 @@ object MyExplicitExceptionHandler {
       }
   }
 
-  object MyApp extends App {
     implicit val system = ActorSystem()
```

Comment:
Can you also fix the indentation?

Reply:
Ah I meant removing that extra space characters in this block (and another one below) that formerly belonged to `MyApp`.